### DISCUSSION TITLE
Remove shortcut for command in amethyst.yml?`

### DISCUSSION THREAD

**Discussion by nbbaier** [ianyh/Amethyst#1488] [Q&A]
Is it possible to remove shortcuts through configuration in .amethyst.yml? For example, I don't really use the "increase-main" command and would like to free up that shortcut key for other things. But the following doesn't remove the shortcut after relaunch, only restore it to default:

```
increase-main:
   - mod:
   - key:
```

Is there a way to do this where my removed shortcuts will persist as removed across relaunches?

**Comment by ianyh** [comment]
Tracked in #1329
